app/containers/ExplorePage.tsx

```diff
@@ -90,19 +156,36 @@ export default function ExplorePage() {
     certaintyRange: [0, 1]
   });
   const [data, setData] = useState<undefined | ObservationsData>();
+  const [inspectedObservations, setInspectedObservations] = useState<
+    Observation[]
+  >([]);
+  const [predictionOverrides, setPredictionOverrides] = useState<
+    Record<string, CreatableOption>
+  >({});
+
   const handleFilters = (val: string[]) => {
     setFilters({ ...filters, ...val });
   };
 
+  const handlePredictionOverride = (
+    location: string,
+    override: CreatableOption
+  ) => {
+    const overrides = { ...predictionOverrides, [location]: override };
+    if (!override) {
```

Comment:
I'm not sure i understand this fully (I don't know about CreatableOption) but looks a bit suspicious and Python-like... In Javascript I'd expect full comparison with `!==`. using just a negation suggests this is a bool but I don't see this in the

Reply:
Good point, refactored the code to make it more clear what's happening.